client/app/nasha/add/nasha-unavailable.html

```diff
@@ -0,0 +1,19 @@
+<div class="cui-page">
+    <oui-page-header
+        data-heading="{{::'nasha_order_title' | translate}}">
+    </oui-page-header>
+
+    <div class="cui-page__content nasha-unavailable">
+        <h2 class="oui-header_3" data-translate="nasha_order_unavailable_subtitle"></h3>
```

Comment:
Using a `h2` tag with `.oui-header_3` as a class is a bit strange. Isn't it?

Reply:
Yes, that wasn't my decision too. CX wanted it to have the style of h3. But I wasn't feeling good putting a h3 without having any h2.

Reply:
As you should be 😃